posts/2017-09-14-serverless-v1.22.0.md

```diff
@@ -0,0 +1,232 @@
+
+serverless-plugin-webpack - A serverless plugin to automatically bundle your functions individually with webpack
+serverless-webpack - Serverless plugin to bundle your lambdas with Webpac
+
+To install a plugin run 'serverless plugin install --name plugin-name-here'
+
+It will be automatically downloaded and added to your package.json and serverless.yml file
+```
+
+After deciding which plugin to use we simply run `serverless install --name serverless-webpack`:
+
+```bash
+root@64d555c6e40a:/app# serverless plugin install --name serverless-webpack
+Serverless: Creating an empty package.json file in your service directory
+Serverless: Installing plugin "serverless-webpack@latest" (this might take a few seconds...)
+Serverless: Successfully installed "serverless-webpack@latest"
+```
+
+That's it. Serverless will do all the heavy lifting behind the scenes (creating an empty `package.json` file (if not yet present), installing the plugin with its peer dependencies, adding the plugin to the `serverless.yml` file, ...). After doing that we're off the races.
+
+If we want to remove a plugin from our service we simply run the `serverless plugin uninstall` command:
+
+```bash
+root@64d555c6e40a:/app# serverless plugin uninstall --name serverless-webpack
+Serverless: Uninstalling plugin "serverless-webpack" (this might take a few seconds...)
+Serverless: Successfully uninstalled "serverless-webpack"
+```
+
+**Note:** Serverless pulls plugin information from the official [`serverless/plugins` repository](https://github.com/serverless/plugins). Make sure to add yours if you want your plugin to be installable via the CLI as well.
+
+**Aside:** The old way of using `npm` to install / uninstall plugins will continue to work as well!
+
+### SSM Parameters in Serverless Variables
+
+AWS SSM Parameters are a great and recommended way to share encrypted variables between and within services.
+
+Serverless v1.22 introduces support for SSM Parameters via the Serverless Variables system.
+
+Let's assume that we've set the following SSM Parameters:
+
+```bash
+aws ssm put-parameter --name foo --value bar --type SecureString
+aws ssm put-parameter --name bar --value foo --type String
+aws ssm put-parameter --name /foo/bar --value bar --type SecureString
+aws ssm put-parameter --name /bar/foo --value foo --type String
+```
+
+We could access those values in your `serverless.yml` file as follows (where `~true` means that the value will be decrypted):
+
+```yml
+service:
+  name: my-service
+
+provider:
+  name: aws
+  runtime: nodejs6.10
+
+custom:
+  foo: ${ssm:foo}
+  bar: ${ssm:bar~true}
+  fooBar: ${ssm:/foo/bar}
+  barFoo: ${ssm:/bar/foo~true}
+  
+# ...snip...
+```
+
+You can read more about the SSM Parameter support in Serverless Variables in the [corresponding documentation](https://serverless.com/framework/docs/providers/aws/guide/variables#reference-variables-using-the-ssm-parameter-store).
+
+### Update function config when running `deploy function`
```

Comment:
You may want to mention this is only for AWS, since Azure & OpenWhisk also implement this cli command

Reply:
Updated!